### PR TITLE
Create craete arifintajul4.json

### PR DESCRIPTION
**Pull request by arifintajul4**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://www.tajul.my.id/

### CODE REVIEW

File: domains/craete arifintajul4.json
Comment (DEV-DIBSTER) [inline domains/craete arifintajul4.json:1]: Invalid file, it cannot have spaces in it's name.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/arifintajul4.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "arifintajul4",
+    "email": "arifintajul4@gmail.com"
+  },
+  "record": {
+    "RECORDTYPE": "tajul.my.id"
```
Comment (DEV-DIBSTER) [inline domains/arifintajul4.json:7]: RECORDTYPE should be an actual record type and not a placeholder.

Please refer to the docs: https://docs.is-a.dev/

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Review by DEV-DIBSTER — APPROVED** [review]

**Comment by arifintajul4** [comment]
still cant access

**Comment by CuteDog5695** [comment]
Did you connect the domain to the hosting provider?

**Comment by arifintajul4** [comment]
Screenshot 2024-10-13 at 18 05 20


> Did you connect the domain to the hosting provider?

**Pull request merged**